modules/packages/Crypto.chpl

```diff
@@ -0,0 +1,876 @@
+/*  A cryptographic library based on OpenSSl, targeted at flexible encryption
```

Comment:
You should add `Crypto.chpl` to `PACKAGES_TO_DOCUMENT` in `modules/Makefile` and make sure it looks good in the generated documentation after doing `make docs`.  The generated documentation goes into `doc/html/index.html`.

Reply:
👍